remote-release-notes/v1_63.md

```diff
@@ -14,6 +14,22 @@ You can now open a folder in a container on a Linux SSH server. First open the f
 
 You will need the Docker CLI installed on the SSH server. Docker is not required locally.
 
+### Preview: User-provided Container Features
+
+A new syntax in the devcontainer.json's features property allows for including container features from a release in a GitHub repository:
+```json
+{
+    "image": "mcr.microsoft.com/vscode/devcontainers/base:bionic",
+    "features": {
+        "codspace/helloworld01/helloworld": true
```

Comment:
@joshspicer Do we have a good example we could use here?

Reply:
The example i've been using to test is https://github.com/codspace/myfeatures, so `codspace/myfeatures/helloworld`.  It adds a program `hello` to your path, that prints `hello, {greeting}` to standard out.  Greeting is whatever you specify in the options.

eg:
```json
	"codspace/myfeatures/helloworld": {
			"greeting": "heyfirst"
		},
```